docs/index.md

```diff
@@ -71,11 +71,12 @@ connection "scaleway" {
   # Set the static credential with the `access_key` and `secret_key` arguments.
   # Alternatively, if no creds passed in config, you may set the environment
   # variables using the `SCW_ACCESS_KEY` and `SCW_SECRET_KEY` arguments.
-  # access_key = "SCWKMH185ZG5THRH7WVX"
-  # secret_key = "ee3b5cb8-2c81-887c-a772-17d46dd34vc7"
-
+  access_key = "YOUR_ACCESS_KEY"
```

Comment:
Can you please revert these changes? We like to include (fake) examples so users know what values can look like.

Reply:
Yes i can revert ;)